app/helpers/alchemy/elements_helper.rb

```diff
@@ -159,7 +159,7 @@ def render_element(element, part = :view, options = {}, counter = 1)
         locals: options.delete(:locals) || {}
       }
 
-      element.store_page(@page) if part.to_sym == :view
+      element.store_page(@page) if part.to_sym == :view && Alchemy::Config.get(:cache_pages)
```

Comment:
We should ask if the `@page` needs to be cached or not. Using the global setting is ok, but will very unlikely turned off for the most sites.